### PR TITLE
Vungle 6.8.0 adapter release

### PR DESCRIPTION
**Pull request by HuFangshuai**
* Verified compatibility with Vungle SDK 6.8.0.
* Vungle's SDK has Android 11 Support (Since 6.7.1), adapter support pending.
* Fixed issue where a bad url could cause an invalid onAdLeftApplication callback.
* Removed FlexFeed and FlexView formats.

### CODE REVIEW

File: ThirdPartyAdapters/vungle/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## Vungle Android Mediation Adapter Changelog
 
+#### Version 6.8.0.0
+- Verified compatibility with Vungle SDK 6.8.0.
+- Vungle's SDK has Android 11 Support (Since 6.7.1), adapter support pending.
```
Comment (ericleich) [inline ThirdPartyAdapters/vungle/CHANGELOG.md:5]: Does the adapter need to do anything different? Do we need this release note?

File: ThirdPartyAdapters/vungle/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+- Fixed issue where a bad url could cause an invalid onAdLeftApplication callback
```
Comment (ericleich) [inline ThirdPartyAdapters/vungle/CHANGELOG.md:6]: I don't see any changes in this PR that would fix this. Is that a Vungle SDK issue that was fixed in 6.8.0? If so, this should be in vungle SDK release notes, not the adapter release notes.

File: ThirdPartyAdapters/vungle/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 ## Vungle Android Mediation Adapter Changelog
 
+#### Version 6.8.0.0
```
Comment (ericleich) [inline ThirdPartyAdapters/vungle/CHANGELOG.md:3]: Should add a note saying that the setFlexViewCloseTimeInSec() method on VungleExtrasBuilder was removed.

File: ThirdPartyAdapters/vungle/CHANGELOG.md (hunk 1 above)
Comment (HuFangshuai) [reply ThirdPartyAdapters/vungle/CHANGELOG.md]: Removed this note.

File: ThirdPartyAdapters/vungle/CHANGELOG.md (hunk 2 above)
Comment (HuFangshuai) [reply ThirdPartyAdapters/vungle/CHANGELOG.md]: yes, it's native sdk changes, removed on adapter side.

File: ThirdPartyAdapters/vungle/CHANGELOG.md (hunk 3 above)
Comment (HuFangshuai) [reply ThirdPartyAdapters/vungle/CHANGELOG.md:3]: Thanks, have added this note.

**Pull request merged**